composer/loggers/mosaicml_logger.py

```diff
@@ -113,6 +114,10 @@ def after_load(self, state: State, logger: Logger) -> None:
                     log.debug(f'Logging WandB run URL to metadata: {run_url}')
                 else:
                     log.debug('WandB run URL not found, not logging to metadata')
+            if isinstance(callback, MLFlowLogger) and callback._enabled:
+                run_url = f'{os.environ["DATABRICKS_HOST"]}ml/experiments/{callback._experiment_id}/runs/{callback._run_id}'
```

Comment:
shouldn't this be an attr on the logger instead of defined here? Also, please use `posixpath.join` to get reliable joining behavior.